### DISCUSSION TITLE
Recommended way to generate GPG key

### DISCUSSION THREAD

**Discussion by fruzyna** [zeapo/Android-Password-Store#2569] [Q&A] [answered]
Any new passwords I generate on my PC are unable to be opened by openkeychain so I decided to give the snapshot builds a try. Unfortunately, I run into #2461 both with my original key and with several new keys I have generated (then re-encrypting my entire database). Is there a recommended set of options to use when generating a new key to avoid the SED packet error?

**Comment by msfjarvis** [comment, marked as answer]
You can change the key preferences after creation to remove AEAD https://unix.stackexchange.com/questions/697579/gpg-failing-with-no-error-for-one-of-my-encrypted-files
